Source/WebCore/workers/service/server/SWServerRegistration.h

```diff
@@ -133,6 +141,8 @@ class SWServerRegistration : public RefCounted<SWServerRegistration>, public Can
     RefPtr<SWServerWorker> m_waitingWorker;
     RefPtr<SWServerWorker> m_activeWorker;
 
+    HashSet<CookieChangeSubscription, CookieChangeSubscriptionHash, SimpleClassHashTraits<CookieChangeSubscription>> m_cookieChangeSubscriptions;
```

Comment:
If done properly, you should not need to specify those 2 last template parameters. You're likely missing the `HashTraits` template specialization for your type. You can use NetworkCacheKey.h as example.